jwt/audience_test.go

```diff
@@ -1,40 +1,40 @@
 package jwt
 
 import (
-	"fmt"
+	"strings"
 	"testing"
 )
 
 func TestAudiences(t *testing.T) {
 	testTable := []struct {
 		name string
 		aud  string
-		err  error
+		err  string
```

Comment:
What was the point of this?

Reply:
The tests were broken, and I changed them to include a message rather than equal a message (out of convenience). Seemed easier to change the expected value to a string.